### PR TITLE
Standardize modal & input error accessibility ids

### PR DESCRIPTION
**Pull request by RyanRory**
This is a PR to standardize the accessibility ids of components in modals and input error message for automation test

### CODE REVIEW

**Review by mpretty-cyro — APPROVED** [review]

File: SessionUIKit/Components/ConfirmationModal.swift [hunk 1]
```diff
@@ -275,9 +275,11 @@ public class ConfirmationModal: Modal, UITextFieldDelegate {
         
         titleLabel.isAccessibilityElement = true
         titleLabel.accessibilityIdentifier = "Modal heading" // stringlint:disable
+        titleLabel.accessibilityLabel = titleLabel.text
         
         explanationLabel.isAccessibilityElement = true
         explanationLabel.accessibilityIdentifier = "Modal description" // stringlint:disable
+        explanationLabel.accessibilityLabel = explanationLabel.text
```
Comment (mpretty-cyro) [inline SessionUIKit/Components/ConfirmationModal.swift:282]: @RyanRory Just wondering with these - wouldn't the screen reader already read out the content of the labels without this? (Or is this change specifically for the automated UI tests?)

Comment (RyanRory) [reply SessionUIKit/Components/ConfirmationModal.swift:282]: This is a change for automated test. As far as I know the screen reader should be able to read out the text, and the `accessibilityLabel` should be set to the text as default, but it seems like it doesn't and Appium can't get the `accessibilityLabel` correctly, so I have to set them explicitly.

Comment (mpretty-cyro) [reply SessionUIKit/Components/ConfirmationModal.swift:282]: Yea no worries - bit of a shame (as ideally we wouldn't custom set this and let the OS handle it) but I guess at the moment we aren't really screen reader accessible so when we eventually take a proper look at that we can change this if it causes issues

**Review by mpretty-cyro — APPROVED** [review]

**Comment by mpretty-cyro** [comment]
@RyanRory I think this should all be fine - just need to merge in dev and resolve the conflict, then feel free to merge 🙂

**Pull request merged**